### PR TITLE
TURTLES-623: Monitor bonding stats/status

### PR DESCRIPTION
**Pull request by tonytan4ever**

### CODE REVIEW

**Comment by tonytan4ever** [comment]
recheck_all

File: playbooks/maas-host-network.yml [hunk 1]
```diff
@@ -52,3 +77,21 @@
     - vars/maas-host.yml
   tags:
     - maas-hosts-network
+
+- name: Install checks for bonding interfaces
+  hosts: hosts
+  gather_facts: false
+  tasks:
+    - name: Install network throughput checks
```
Comment (npawelek) [inline playbooks/maas-host-network.yml]: Probably should update this play's name to reflect

File: playbooks/files/rax-maas/plugins/bonding_iface_check.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+from maas_common import metric_bool
+from maas_common import print_output
+
+import netiface_check_lib
+
+
+def bonding_ifaces_check():
+    bonding_ifaces = os.lisdir("/proc/net/bonding")
+    for bonding_iface in bonding_ifaces:
```
Comment (npawelek) [inline playbooks/files/rax-maas/plugins/bonding_iface_check.py]: I've noticed that some environments might have a bond file located there, however, it's empty and contains no sub-interfaces. Will this fail to iterate over causing an alarm or just negate the output altogether?

Comment (tonytan4ever) [reply playbooks/files/rax-maas/plugins/bonding_iface_check.py]: This would just produce no metric (skip the iteration) and generate no alarns

**Comment by tonytan4ever** [comment]
recheck_cit_trusty_liberty_deploy

**Comment by tonytan4ever** [comment]
recheck_cit_trusty_liberty_deploy

**Comment by npawelek** [comment]
@tonytan4ever Can we get this merged today?

**Pull request merged**